sdk/monitor/azingest/autorest.md

```diff
@@ -6,7 +6,6 @@ description: Azure Monitor Ingestion Go Client
 generated-metadata: false
 
 clear-output-folder: false
-export-clients: true
```

Comment:
Any particular reason to remove this line? Noticed it was left in the other modules.

Reply:
It's a vestigial setting from older versions of the code generator. Doesn't hurt anything but doesn't do anything either. We should remove it everywhere at some point.

Reply:
I updated the PR for `azquery` to remove it there too.